app/code/BVMyBlog/Blog/Controller/Adminhtml/Post/Upload.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+namespace BVMyBlog\Blog\Controller\Adminhtml\Post;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+
+/**
+ * Class Upload
+ *
+ * Controller Upload image
+ */
+class Upload extends \Magento\Backend\App\Action implements HttpGetActionInterface
+{
+    /**
+     * Image uploader
+     *
+     * @var \Magento\Catalog\Model\ImageUploader
+     */
+    protected $imageUploader;
+
+    /**
+     * Upload constructor.
+     *
+     * @param \Magento\Backend\App\Action\Context $context
+     * @param \Magento\Catalog\Model\ImageUploader $imageUploader
+     */
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Catalog\Model\ImageUploader $imageUploader
+    ) {
+        parent::__construct($context);
+        $this->imageUploader = $imageUploader;
+    }
+
+    /**
+     * Upload file controller action.
+     *
+     * @return \Magento\Framework\Controller\ResultInterface
+     */
+    public function execute()
+    {
+        $imageUploadId = $this->_request->getParam('param_name', 'review_image');
```

Comment:
Weird parameter name - **param_name**. Also why default value is **review_image**?